### PR TITLE
Include NSLayoutConstraint+FluidConstraint.swift

### PR DESCRIPTION
**Pull request by renatosarro**

### CODE REVIEW

File: Pods/Classes/NSLayoutConstraint+FluidConstraint.swift [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+//  Copyright © 2017 Renato Matos. All rights reserved.
+//
+
+import Foundation
+
+extension NSLayoutConstraint {
+    
+    static func fluidConstraints(arrViews: [UIView], alignAxis: String, vMargin: CGFloat, hMargin: CGFloat, iMargin: CGFloat) -> [NSLayoutConstraint] {
+        
+        let arrConstraints: NSMutableArray = NSMutableArray(capacity: arrViews.count)
```
Comment (fggeraissate) [inline Pods/Classes/NSLayoutConstraint+FluidConstraint.swift]: Please change let to var and NSMutableArray to [NSLayoutConstraint]

File: Pods/Classes/NSLayoutConstraint+FluidConstraint.swift [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        let dictViews: NSMutableDictionary = NSMutableDictionary()
```
Comment (fggeraissate) [inline Pods/Classes/NSLayoutConstraint+FluidConstraint.swift]: Please change let to var and NSMutableDictionary to Dictionary<String,UIView>

File: Pods/Classes/NSLayoutConstraint+FluidConstraint.swift [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            stringAppended = "|"
+        } else {
+            stringAppended = String(describing: isVertical ? vMargin : hMargin) + "-|"
+        }
+        
+        stringGlobalFormat.append(stringAppended)
+        
+        arrConstraints.addObjects(from: NSLayoutConstraint.constraints(withVisualFormat: stringGlobalFormat as String, options: NSLayoutFormatOptions(rawValue: 0), metrics: nil, views: dictViews as! [String : Any]))
+        
+        return arrConstraints as! [NSLayoutConstraint]
```
Comment (fggeraissate) [inline Pods/Classes/NSLayoutConstraint+FluidConstraint.swift]: Please remove these force unwraps :)

File: Pods/Classes/NSLayoutConstraint+FluidConstraint.swift (hunk 1 above)
Comment (renatosarro) [reply Pods/Classes/NSLayoutConstraint+FluidConstraint.swift]: Done.

File: Pods/Classes/NSLayoutConstraint+FluidConstraint.swift (hunk 2 above)
Comment (renatosarro) [reply Pods/Classes/NSLayoutConstraint+FluidConstraint.swift]: Done.

File: Pods/Classes/NSLayoutConstraint+FluidConstraint.swift (hunk 3 above)
Comment (renatosarro) [reply Pods/Classes/NSLayoutConstraint+FluidConstraint.swift]: Done.

**Review by fggeraissate — APPROVED** [review]

**Pull request merged**